local-registry/Cargo.toml

```diff
@@ -81,36 +71,26 @@ eval = "0.4.3"
 evalexpr = "11.0.0"
 factor = "0.4.0"
 factorial = "0.3.0"
-failure = "0.1.8"
-failure_derive = "0.1.8"
 fancy-regex = "0.11.0"
 fasteval = "0.2.4"
 fixedbitset = "0.4.2"
 flagset = "0.4.3"
 flame = "0.2.2"
 flamer = "0.5.0"
-fluent = "0.16.0"
 fnv = "1.0.7"
 foreach = "0.3.0"
 free-ranges = "1.0.7"
 frunk = "0.4.2"
-futures-core = "0.3.28"
```

Comment:
Is this needed for futures?

Reply:
Yeah, this and a couple other async-related crates are maybe the most risky crates I removed. futures are used extensively in the Rust ecosystem, but only for IO related tasks, which are not relevant to our exercises.

It would be a needless complication to solve our exercises by wrapping the solution in a future and then executing it with an executor.

Reply:
Okay. Just keep in mind that there are exercises that deal with IO, so when those will be added we should probably enable this.